Fixed #45

TheCreativeGod opened the pull request:
Cuando un usuario está stremeando, se le asigna el rol `[twitch]`

TheCreativeGod committed:
Fixed #45

Cuando un usuario está stremeando, se le asigna el rol `[streamer]`

pagoru commented:
Realmente es solo para darle visibilidiad momentania al que esta haciendo stream, una vez deje de hacerlo, el rol se le deberia quitar.
Quizá:
- Que al inicio del bot se quite a todo el mundo de ese rol
- Que se compruebe si hay alguien en modo stream al iniciar el bot.
- Que se detecte cuando los usuarios dejan de estar en ese modo i cuando lo estan.

Creo que seria mas útil, dejar de forma permanente a alguien no, simplemente es poderles dar la oportunidad que la gente pueda ver que hay alguien en stream. 👍

TheCreativeGod committed:
Role name, role remove.

- Detectar cuando el usuario deja de stremear
- Cambio del nombre del rol, de `[streamer]` a `[streaming]`

TheCreativeGod committed:
Merge branch 'develop' into tcg-streaming

pagoru committed:
Update presenceUpdate.js

pagoru commented:
Siguen faltando los puntos 
- Que al inicio del bot se quite a todo el mundo de ese rol
- Que se compruebe si hay alguien en modo stream al iniciar el bot.

En caso de caída del bot mientras alguien sigue en streaming, seguirá con el rango hasta que vuelva a hacer streaming.
